packages/design-tokens/src/tokens/functional/size/viewport.json

```diff
@@ -1,27 +1,27 @@
 {
   "brand": {
     "viewportRange": {
-      "narrow": {
+      "narrow-viewport": {
```

Comment:
@rezrah this is added to be able to replace your custom media format. 

However, I think it would be better to just remove the `-viewport` part. You already have `viewportRange` before the name, so this should be enough.

Reply:
That seems reasonable @lukasoppermann. We'd need to just add a changeset to indicate this change. 

Ideally we just grab this file from `primer/primitives` though, is that still the plan and just out of scope for this PR?

Reply:
Yes, this would be optimal.